_data/collection-index.yml

```
@@ -668,3 +668,8 @@
   contact: https://github.com/rsm-hcd
   repository: https://github.com/rsm-hcd/devcontainer-templates
   ociReference: ghcr.io/rsm-hcd/devcontainer-templates
+- name: Additional Dev Container Features by Georg Ofenbeck 
+  maintainer: Georg Ofenbeck
+  contact: https://github.com/GeorgOfenbeck/features/issues
+  repository: https://github.com/GeorgOfenbeck/features 
+  ociReference: ghcr.io/GeorgOfenbeck/features
```

```suggestion
  ociReference: ghcr.io/georgofenbeck/features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

---

of course! thanks